org.mwc.debrief.ui_test/test-cases/view_and_edit_some_data/create_new_plot.test

```diff
@@ -50,7 +50,7 @@ with [get-window -class WizardDialog] {
 set-dialog-result Color "255,0,128"
 with [get-window -class WizardDialog] {
     get-button Color | click
-    get-combo -after [get-label "Delta:"] | select [get-by-os -linux "1 km" -macosx "1.0 km" -default "1 km" ] // on mac (1.0 )
+    get-combo -after [get-label "Delta:"] | select [get-by-os -linux "1 km" -macosx "1 km" -default "1 km" ] // on mac (1.0 )
```

Comment:
@theanuradha - on my mac, we no longer need O/S specific values for this drop-down.   Well, not now that I've merged the `develop` branch back in.

Could you please test this on your machine?  

If your mac works with `1 km`, could you remove the O/S specific way of doing this, please.

Reply:
@IanMayo sure